site/src/atomics/spacing.md

```diff
@@ -6,7 +6,70 @@ template: standard
 
 # Spacing Atomics
 
+Spacing atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+| cssproperty                                                                                                                                                                                                            | value                                                   | screensize                        |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- | --------------------------------- |
+| `margin`, `padding`, `margin-block`, `padding-block`,`margin-inline`, `padding-inline`, `margin-top`, `padding-top`, `margin-right`, `padding-right`, `margin-bottom`, `padding-bottom`, `margin-left`, `padding-left` | `xxs`, `xs`, `s`, `m`, `l`, `xl`, `xxl`, `xxxl`, `none` | `tablet`, `desktop`, `widescreen` |
+
+## Usage
+
+_Adding uniform spacing_
+
+Use `margin` or `padding` with a size value to add spacing around an element.
```

Comment:
Remove _Adding uniform spacing_?

Use `margin` or `padding` with a size value to add uniform spacing around an element.

Reply:
Updated